server/src/main/java/org/jetbrains/bsp/bazel/server/sync/BazelProjectMapper.java

```diff
@@ -139,4 +141,12 @@ private Map<URI, Label> buildReverseSourceMapping(Seq<Module> modules) {
         });
     return HashMap.ofAll(output);
   }
+
+  private Map<String, String> environmentItem(TargetInfo target) {
+    if (target.containsEnvInherit("80")) {
```

Comment:
what is "80"?

Reply:
it is the number I assigned to env_inherit in bsp_target_info.proto